spec/models/submission_spec.rb

```diff
@@ -12,4 +14,10 @@
       expect(Submission.unsent).not_to eql nil
     end
   end
-end
\ No newline at end of file
+
+  describe '#enqueue' do
+    it 'sent into delivery' do
+
+    end
```

Comment:
I didn't know what to do with the existing specs. AFAIK we write specs for the public methods, and both 'valid' and 'sent' aren't methods in `submission.rb`. 
But what should be tested in the methods that _are_ present? 🤔